File: QRCode.Core/Domain/qr_enterprise.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace QRCode.Core.Domain
{
    public class qr_enterprise
    {
        public virtual long qrenterpriseid { get; set; }
        public virtual string name { get; set; }
        public virtual string taxcode { get; set; }
        public virtual string tel { get; set; }
        public virtual string email { get; set; }
        public virtual string fax { get; set; }
        public virtual string logo { get; set; }
        public virtual string nation { get; set; }
        public virtual string province { get; set; }
        public virtual string district { get; set; }
        public virtual string wards { get; set; }
        public virtual string occupation { get; set; }
        public virtual string additional { get; set; }
        public virtual DateTime? created_date { get; set; }
        public virtual int created_by { get; set; }
        public virtual DateTime? lastcreated_date { get; set; }
        public virtual int lastcreated_by { get; set; }
        public virtual bool status { get; set; }
        public virtual string address { get; set; }

    }
}

File: QRCode.FEW/ClientApp/src/app/models/qr_enterprise.ts
import { productview } from "./product";

export interface qr_enterprise {
    qrenterpriseid: number;
    name: string;
    taxcode: string;
    tel: string;
    email: string;
    fax: string;
    logo: string;
    nation: string;
    province: string;
    district: string;
    wards: string;
    occupation: string;
    additional: string;
    created_date: Date;
    created_by: number;
    lastcreated_date: Date;
    lastcreated_by: number;
    status: boolean;
    address: string;
    sectors_code: string;
    url_background: string;
    url_video: string;
    url_img: string
}
export interface enterprisview {
    additional: string;
    address: string;
    email: string;
    fax: string;
    logo: string;
    name: string;
    qrenterpriseid: number;
    tel: string;
    list_ref: productview[]
}
export interface qr_enterprise_excel {
    name: string;
    taxcode: string;
    tel: string;
    email: string;
    fax: string;
    logo: string;
    nation: string;
    province: string;
    district: string;
    wards: string;
    occupation: string;
    additional: string;
    address: string;
    sectors_code: string;
    url_background: string;
    url_video: string;
    url_img: string;
    err_str:string;
    Website:string;
    Facebook:string;
    Shopee:string;
    Zalo:string;
    Instagram:string;
    Tiktok:string;
    Tiki:string;
    Youtube:string;
    Linkedin:string;
    Lazada:string;
    Sendo:string;
}